apps/admin/backend/src/cast_vote_records.ts

```diff
@@ -81,6 +100,79 @@ function validateCastVoteRecordAgainstElectionDefinition(
   return ok();
 }
 
+/**
+ * Lists the cast vote record exports on the inserted USB drive
+ */
+export async function listCastVoteRecordExportsOnUsbDrive(
```

Comment:
I copied this function out of `legacy_cast_vote_records.ts` (deleted below, in this commit), since it's still needed.

The main edit that I made after copying it out of `legacy_cast_vote_records.ts` was removing the code for old-format CVR exports (and corresponding feature-flag branching). I also removed the logging inside the function since, in a subsequent commit, I move that logging up to `app.ts`.